src/screens/population-scale/index.js

```diff
@@ -0,0 +1,58 @@
+import { motion, useScroll, useTransform } from "framer-motion";
+import { useRef } from "react";
+import { FarmedAnimals } from "./farmed-animals";
+
+export function PopulationScale() {
+  const scrollRef = useRef(null);
+  const { scrollYProgress } = useScroll({
+    target: scrollRef,
+    offset: ["start start", "end end"],
+  });
+  const earthOpacity = useTransform(scrollYProgress, [0, 1], [1, 0.3]);
+  const humanOpacity = useTransform(scrollYProgress, [0, 0.75], [1, 0]);
+  const animalsOpacity = useTransform(scrollYProgress, [0.75, 1], [0, 1]);
+
+  return (
+    <div ref={scrollRef} className="min-h-screen w-full bg-abyss bg-cover">
+      <div className="mx-auto grid min-h-screen w-full max-w-7xl grid-cols-2 gap-10 px-10">
+        <div className="flex flex-col">
+          <div className="grid h-screen place-items-center">
+            <h2 className="flex flex-col font-brand text-5xl leading-tight text-white">
+              <span>There are roughly</span>
+              <span className="text-blue-200">8 billion humans</span>
+              <span>on planet Earth.</span>
+            </h2>
+          </div>
+          <div className="grid h-screen place-items-center">
+            <h2 className="flex flex-col font-brand text-5xl leading-tight text-white">
+              <span>For every human, there are roughly</span>
+              <span className="text-blue-200">3-4 farmed animals</span>
+              <span>being farmed at any given moment.</span>
+            </h2>
+          </div>
+        </div>
+        <div className="sticky top-0 grid h-screen place-items-center">
+          <div>
+            <motion.img
+              style={{ opacity: earthOpacity }}
+              src="/images/population-scale/earth.png"
+              alt=""
+            />
+            <motion.img
+              style={{ opacity: humanOpacity }}
+              className="absolute inset-0 left-1/2 top-1/2 w-1/4 -translate-x-1/2 -translate-y-1/2"
+              src="/images/population-scale/human.png"
```

Comment:
I'm curious about this image src. Usually I have seen it more like 
`import human from ../assets/human.png` and then `src={human}`.  Mostly wondering about our folder structure because it seems like we might have some images in `assets` and some in `public` now. Should we just one?

Reply:
Right now it's still a mix, but my goal was to move everything to public eventually. Unless there's a good reason why we should stick to importing images with Webpack. One thing that comes to mind is image optimization, but there are tools out there that work with publicly hosted images as well.

Reply:
cool makes sense